docs/models/nuclear/hydrogen_tank.rst

```diff
@@ -1,37 +1,52 @@
 Hydrogen Tank
 ================
 
-The DISPATCHES Hydrogen Tank Model
+The DISPATCHES Hydrogen Tank Model represents a steady state implementation of a compressed hydrogen gas tank. This tank model supports tank filling and emptying operations for a fixed duration assuming a constant flow.
 
 Degrees of Freedom
 ------------------
 
-The Hydrogen Tank Model has X degrees of freedom.
+The Hydrogen Tank Model has 9 degrees of freedom. The degrees of freedom include the inlet state variables (flow, pressure, and temperature), previous state variables (temperature and pressure), tank dimensions (tank diameter and tank length), time duration, and outlet flow (for tank filling scenario flow = 0). By default, the tank is set to be adiabatic by fixing heat duty = 0.
```

Comment:
It is hard to give a fixed number of degrees of freedom, as this depends on the property package used. I would suggest that it should be X+inlet DoF to allow for property packages with different specification.